radix_sort.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+
+def radixsort(_list):
+    slots = 10
+    max_length = False
+    holder, placement = -1, 1
+
+    for i, n in enumerate(_list):
+        if n < 0:
+            raise ValueError('only accepts postive integers')
+        _list[i] = int(n)
```

Comment:
It's generally considered a bad idea to mutate the data type of items in an object passed in to your function.  If you intend to handle non-integers in the incoming list by casting them to integers, you probably want to make a copy of the incoming list first.  That way, the user's data is not changed without their knowing it.